modules/apps/list-type/list-type-service/src/main/java/com/liferay/list/type/service/impl/ListTypeEntryLocalServiceImpl.java

```diff
@@ -176,6 +215,22 @@ private void _validateExternalReferenceCode(
 		}
 	}
 
+	private void _validateInvokerBundle(long listTypeDefinitionId)
```

Comment:
What do you think about creating a `ListTypeDefinitionUtil` class and putting this method there, this way we can reuse this code in both services.